OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -614,12 +614,12 @@ public override bool Tick(Actor self)
 
 				// Reserve the exit cell
 				mobile.SetPosition(self, cell, subCell);
-				mobile.SetVisualPosition(self, pos);
 
+				// mobile.SetVisualPosition(self, self.CenterPosition);
```

Comment:
Why is this commented out?

Reply:
I didn't test the new changes and wasn't sure if this could go away.